src/main.cpp

```diff
@@ -83,11 +87,16 @@ int mainWhichMayThrowExceptions(int argc, char** argv) {
    args = {args.begin() + 1, args.end()};
    if (mode_argument == "preprocessing") {
       mode = ExecutionMode::PREPROCESSING;
+   } else if (mode_argument == "append") {
+      mode = ExecutionMode::APPEND;
    } else if (mode_argument == "api") {
       mode = ExecutionMode::API;
+   } else if (mode_argument == "initialize") {
+      mode = ExecutionMode::INITIALIZE;
    } else {
       std::cerr << program_name
-                << ": need either 'preprocessing' or 'api' as the first program argument, got '"
+                << ": need 'preprocessing', 'initialize', 'append' or 'api' as the first program "
+                   "argument, got '"
```

Comment:
```
$ build/Debug/silo
silo: need either 'preprocessing' or 'api' as the first program argument
``` 
Let's do the same in lin 82 (or extract that message into a commonly used variable/method)

Reply:
`config_specification.cpp`, line 66 has the same issue (because it only says "use 'silo api|preprocessing --help'")